src/resolver/findAllComponentDefinitions.js

```diff
@@ -34,7 +35,18 @@ export default function findAllReactCreateClassCalls(
     return false;
   }
 
+  function statelessVisitor(path) {
+    if (isStatelessComponent(path)) {
+      // TODO: normalizeStatelessDefinition to pick up propTypes
```

Comment:
This is done right? (because we use a different way to resolve this information)